### ISSUE TITLE
additionalProperties not supported

### ISSUE THREAD

**Issue by jeffreytanz** [fotinakis/swagger-blocks#77]
See http://swagger.io/specification/ 

swagger-blocks supports
- items
- allOf
- properties

Swagger-blocks does not support
- additionalProperties

**Comment by fotinakis** [comment]
Yup that looks right, we need to add that. If you need a workaround right now you can specify arbitrary properties with `key`, like:

```ruby
key :additionalProperties, {
  '$ref' => '#/definitions/ComplexModel',
}
```
